paramiko/ssh_exception.py

```diff
@@ -84,9 +82,7 @@ def __init__(self, types):
         self.allowed_types = types
 
     def __str__(self):
-        return "Partial authentication; allowed types: {!r}".format(
-            self.allowed_types
-        )
+        return "Partial authentication; allowed types: {!r}".format(self.allowed_types)
```

Comment:
```
./paramiko/ssh_exception.py:85:80: E501 line too long (87 > 79 characters)
```

Reply:
Same, don’t know what happened here. Will fix it